src/spacyMatcher.py

```diff
@@ -1,54 +1,49 @@
-## Function: spacyMatcher
```

Comment:
This type of header comment really doesn't look very pythonic. Doc strings are usually enclosed in """ multi-line comments, not lots of # lines. 
Also, the comment at the top should document the module. Docstrings for each function should be inside the functions they are documenting

Reply:
Also, no need to put the author and date in a header like this. Git can tell me that information and it's unlikely in "real" work that any single file could really be attributed only to one person as usually many people will contribute to it.

Reply:
David, thanks very much, really appreciate your comments.
I will update / improve the code ;)